core/controllers/base.py

```diff
@@ -503,7 +503,7 @@ def handle_exception(self, exception, unused_debug_mode):
                 self.redirect(user_services.create_login_url(self.request.uri))
             return
 
-        logging.error(b''.join(traceback.format_exception(*sys.exc_info())))
+        logging.exception(b''.join(traceback.format_exception(*sys.exc_info())))
```

Comment:
@vojtechjelinek Do we need traceback.format_exception here? [Or is the line that is printing traceback for all the errors?]

Reply:
Can you confirm what description to be added here

Reply:
Done

Reply:
Replacing the stack trace with 
`logging.exception('Exception raised: %s', exception)`

brings failure in other tests how do I update this

Reply:
You'll have to change the expected error message in the test file to support this new functionality.

Reply:
Done